configure.ac

```diff
@@ -174,7 +174,7 @@ AS_IF([test "x$enable_unit" != xno], [
     AS_IF([test $BASH_SHELL = no],
           [AC_MSG_ERROR([Required executable bash not found, system tests require a bash shell!])])
 
-    AM_PATH_PYTHON([2.7],
+    AM_PATH_PYTHON([],
```

Comment:
@williamcroberts is this related to this fix?

Reply:
It's a separate commit that happens to be in the PR, we don't need to backport it.